hanadb_exporter/exporters/prometheus_exporter.py

```diff
@@ -51,8 +53,11 @@ def _manage_gauge(self, metric, formatted_query_result):
                 except ValueError: # Received data is not a label, check for the lowercased value
                     if column_name.lower() == metric.value.lower():
                         metric_value = column_value
-
-            metric_obj.add_metric(labels, metric_value)
+            if metric_value != None:
```

Comment:
Maybe we should check as well that all the labels were found, not just the value. What do you think?

Reply:
a part of this, personally I would write it more explicit (without if/else)

```
  if metric_value == None:
    log.error("sulla testa o una musica che fa: badabumchacha!")
    continue
```

```